build/mvn

```diff
@@ -67,17 +67,27 @@ install_app() {
   fi
 }
 
-# Install maven under the build/ folder
+# Determine the Maven version from the root pom.xml file and
+# install maven under the build/ folder if needed.
 install_mvn() {
-  local MVN_VERSION="3.3.9"
-  local APACHE_MIRROR=${APACHE_MIRROR:-'https://www.apache.org/dyn/closer.lua?action=download&filename='}
+  local MVN_VERSION=`grep "<maven.version>" "${_DIR}/../pom.xml" | \
+                     head -1 | cut -f2 -d'>' | cut -f1 -d'<'`
```

Comment:
It works as the Jenkins runs.
```
spark:SPARK-14867$ grep "<maven.version>" pom.xml 
    <maven.version>3.3.9</maven.version>
```

For simplification, I'm not sure since I borrowed it from the following `install_scala()` code in the same script. Should I change them both for consistency?
```bash
install_scala() {
  # determine the Scala version used in Spark
  local scala_version=`grep "scala.version" "${_DIR}/../pom.xml" | \
                       head -1 | cut -f2 -d'>' | cut -f1 -d'<'`
```
Please let me know. If needed, I will update both.